Yona/Yona/MeDashBoard/YonaUserProfileViewController.swift

```diff
@@ -246,34 +246,30 @@ class YonaUserProfileViewController: UIViewController, UITableViewDelegate, UITa
     }
     
     func uploadUserData() {
-         UserRequestManager.sharedInstance.updateUser((aUser?.userDataDictionaryForServer())!, onCompletion: {(success, message, code, user) in
+        UserRequestManager.sharedInstance.updateUser((aUser?.userDataDictionaryForServer())!, onCompletion: {(success, message, code, user) in
             //success so get the user?
-                if success {
-                    Loader.Hide()
-                    self.aUser = user
-                    if let _ = user?.confirmMobileLink {
-                        if let controller : ConfirmMobileValidationVC = R.storyboard.login.confirmPinValidationViewController(()) {
-                            controller.isFromUserProfile = true
-                        
-                            self.navigationController?.pushViewController(controller, animated: true)
-                        }
-                    }
-                    self.tableView.reloadData()
-                } else {
-//                    NSLog("----------------------- YONA")
-//                    NSLog("----------------------- uploadUserData")
-//                    NSLog(" ")
-//                    NSLog("           ")
-//                    NSLog("message %@",message!)
-
-                    Loader.Hide()
-                    if let alertMessage = message,
-                        let code = code {
-                        self.displayAlertMessage(code, alertDescription: alertMessage)
+            if success {
+                Loader.Hide()
+                self.aUser = user
+                if let _ = user?.confirmMobileLink{
+                    UserDefaults.standard.set(user?.confirmMobileLink, forKey: YonaConstants.nsUserDefaultsKeys.confirmMobileLink)
```

Comment:
This statement stores(e.g link) the given data as key-value pairs to persist across app launches

Link is stored to address following scenario,
- User joins with mobile number and quits the application before confirming it with OTP.

Also am updating the code to store entire user data instead of link.

Reply:
👍